### PR TITLE
[SLAM] Enable loadOccupancyMap interface

### LINKED ISSUE #72: [SLAM] support loadOccupancyMap
support _loadOccupancyMap_()

### PR DESCRIPTION
**Pull request by lisa0314**
Fix #72 
@DonnaWuDongxia , please help to review.

### CODE REVIEW

File: src/slam/examples/internal-sample.js [hunk 1]
```diff
@@ -70,8 +70,8 @@ function instanceHandler(slamInstance) {
     console.log('error:' + e);
   });
 
-  function saveOccupancyMapAsPpm() {
-    slamInstance.saveOccupancyMapAsPpm( __dirname + '/map.ppm', false)
+  function saveOccupancyMap() {
+    slamInstance.saveOccupancyMap( __dirname + '/map.ppm', false)
```
Comment (DonnaWuDongxia) [inline src/slam/examples/internal-sample.js]: Please create another test file to test the new API. Or you can create the test case in the test folder.

**Comment by DonnaWuDongxia** [comment]
Looks good to me on the implementation. Please make a separate internal test file or add a test case under `test` folder, and do remember to update the doc.

**Comment by DonnaWuDongxia** [comment]
Another reminder, please close the related bug if needed.

**Comment by lisa0314** [comment]
@DonnaWuDongxia , all done.

File: src/slam/examples/internal-sample.js [hunk 2]
```diff
@@ -80,8 +80,10 @@ function instanceHandler(slamInstance) {
       console.log('error:' + e);
     });
   }
-
+
```
Comment (DonnaWuDongxia) [inline src/slam/examples/internal-sample.js]: no space for blank lines.

File: src/slam/examples/internal-sample.js [hunk 3]
```diff
@@ -80,8 +80,10 @@ function instanceHandler(slamInstance) {
       console.log('error:' + e);
     });
   }
-
+  
   setTimeout(saveOccupancyMapAsPpm, 10000);
+  setTimeout(saveOccupancyMap, 10000);
+  setTimeout(loadOccupancyMap, 15000);
```
Comment (DonnaWuDongxia) [inline src/slam/examples/internal-sample.js]: If you have another internal test file, why do we need to add above two lines?

**Comment by DonnaWuDongxia** [comment]
Better! Please remember to update the doc.

File: doc/spec/bikeshed/slam.bs [hunk 4]
```diff
@@ -189,7 +189,6 @@ interface Instance {
 :: *Return type*:Promise&lt;<a>String</a>&gt;
 
 : {{loadOccupancyMap}}
-:: Note: not supported yet.
```
Comment (DonnaWuDongxia) [inline doc/spec/bikeshed/slam.bs:192]: Another "not supported yet" at line-90.
https://github.com/lisa0314/node-realsense-1/blob/120f006ebf63c6a0b3fa0349748bf3444b60db55/doc/spec/bikeshed/slam.bs#L90

**Comment by DonnaWuDongxia** [comment]
And the try-bot issue.

**Comment by halton** [comment]
@DonnaWuDongxia does this PR have your LGTM?

**Comment by DonnaWuDongxia** [comment]
@halton Need another change. @lisa0314 is working on it.

**Comment by DonnaWuDongxia** [comment]
@halton It's LGTM now.

**Pull request merged**